docs/doc_tutorial/advanced/training_seo.rst

```diff
@@ -0,0 +1,135 @@
+---
+SEO
+---
+
+ViUR offers the possibility to make URLs SEO-friendly in multiple languages.
+For this you can define seo-identifier for each url component.
+
+
+Module
+------
+The modulename is the first part of an URL.
+SEO-Identfiers has to be set as class-attribute ``seoLanguageMap`` of type ``dict[str, str]`` in the module.
+It maps a *language* to the acording *identifier*.
+
+.. code-block:: python
+    :name: module seo-map
+    :caption: modules/myorders.py
+    :emphasize-lines: 4-7
+
+    from viur.core.prototypes import List
+
+    class MyOrders(List):
+        seoLanguageMap = {
+            "de": "bestellungen",
+            "en": "orders",
+        }
+
+By default the module would be available under */myorders*, the lowercase modulename.
+With the defined :attr:`seoLanguageMap`, it's now available under */de/bestellungen* and */en/orders*.
+
+Great, this part is now user and robot friendly :)
+
+
+Method
+------
+The methodname is usually the second part of an URL.
+SEO-Identfiers can be provided to the :meth:`exposed<core.exposed>` decorator of type ``dict[str, str]``.
+It maps a *language* to the acording *identifier*.
+
+.. code-block:: python
+    :name: method seo-map
+    :caption: modules/myorders.py
+    :emphasize-lines: 10-13
+
+    from viur.core.prototypes import List
+    from viur.core import exposed
+
+    class MyOrders(List):
+        seoLanguageMap = {
+            "de": "bestellungen",
+            "en": "orders",
+        }
+
+        @exposed({
+            "de": "warenkorb",
+            "en": "cart",
+        })
+        def view_the_cart(self):
+            ...
+
+By default the method would be available under */myorders/view_the_cart*.
```

Comment:
```suggestion
By default, the method would be available under */myorders/view_the_cart*.
```

Reply:
I don't think the comma is correct there. It would be if it was a German sentence...